src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/ui_get_capabilities_response.cc

```diff
@@ -73,9 +73,24 @@ void UIGetCapabilitiesResponse::Run() {
         msg_params[hmi_response::soft_button_capabilities]);
   }
 
-  if (msg_params.keyExists(strings::audio_pass_thru_capabilities)) {
+  // use newer parameter "audioPassThruCapabilitiesList" when available
+  if (msg_params.keyExists(strings::audio_pass_thru_capabilities_list)) {
     hmi_capabilities.set_audio_pass_thru_capabilities(
-        msg_params[strings::audio_pass_thru_capabilities]);
+        msg_params[strings::audio_pass_thru_capabilities_list]);
+  } else if (msg_params.keyExists(strings::audio_pass_thru_capabilities)) {
```

Comment:
Sorry for the late comment, but in this RAI response `msg_params[strings::audio_pass_thru_capabilities]` will always be an object as defined in the `HMI_API`. If you wouldn't mind just removing the case for processing it as an array we will be ready to merge. Thank you!